src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all $time;
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      color: #34568b;
+      transform: scale(1.2);
+      cursor: pointer;
```

Comment:
Move `cursor: pointer;` from hover

Reply:
where move cursor? don't und...  are just remove it?